src/test/java/teammates/test/cases/browsertests/FeedbackConstSumOptionQuestionUiTest.java

```diff
@@ -235,6 +236,57 @@ public void testEditQuestionAction() throws Exception {
         assertEquals("200", feedbackEditPage.getConstSumPointsForEachRecipientBox(1));
     }
 
+    /**
+     * Tests Min and Max Points Constraint.
+     * Element {@code constSumPointsMin} is enabled when Element {@code constSumPointsHasMaxPointsConstraint} is checked.
+     */
+    public void testEditConstSumQuestionWithPointsConstraint() {
```

Comment:
Test add new question?
For end-to-end test, just test one success and failure scenario respectively should be enough as we have put efforts in the integration test.